### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by EmmaDawsonDev** [luisFilipePT/github-covid-finder#44]
Many of the elements are lacking accessibility features like aria labels and roles. Would it be ok if I did some accessibility enhancements on the code?
